### PR TITLE
Feature/todo roland

### PR DESCRIPTION
**Pull request by rscircus**
Basic outline of what there is to do and switching on the vacuum cleaner.

### CODE REVIEW

File: include/layer.hpp [hunk 1]
```diff
@@ -306,17 +303,17 @@ class ConvLayer : public Layer {
 
   void applyBWD(MyReal *state, MyReal *state_bar, int compute_gradient);
 
-  inline MyReal
-  apply_conv(MyReal *state,   // state vector to apply convolution to
-             int output_conv, // output convolution
-             int j,           // row index
-             int k);          // column index
+  inline MyReal apply_conv(
```
Comment (steffi7574) [inline include/layer.hpp:306]: Why the style change for these inline functions?

File: src/network.cpp [hunk 2]
```diff
@@ -35,13 +35,14 @@ void Network::createNetworkBlock(int StartLayerID, int EndLayerID,
   nlayers_local = endlayerID - startlayerID + 1;
   nlayers_global = config->nlayers;
   nchannels = config->nchannels;
-  dt = (config->T) / (MyReal)(config->nlayers - 2); // nlayers-2 = nhiddenlayers
+  dt =
```
Comment (steffi7574) [inline src/network.cpp]: Why do you insert a new line?

File: include/layer.hpp (hunk 1 above)
Comment (rscircus) [reply include/layer.hpp:306]: https://google.github.io/styleguide/cppguide.html#Namespaces

File: src/network.cpp (hunk 2 above)
Comment (rscircus) [reply src/network.cpp]: Thanks. That one is stupid.

**Review by rscircus — COMMENTED** [review]
Ok, so far I'd say this is good. Please merge this pull request, if you are happy with that one.

File: include/layer.hpp (hunk 1 above)
Comment (rscircus) [reply include/layer.hpp:306]: Basically that's the style guide I'm following. There is an example in this section and an explicit section on inline functions.

**Pull request merged**